service/src/main/java/uk/gov/hmcts/reform/fpl/validation/validators/time/EPOTimeRangeValidator.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.fpl.validation.validators.time;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.validation.interfaces.time.EPOTimeRange;
+import uk.gov.hmcts.reform.fpl.validation.interfaces.time.TimeDifference;
+
+import java.time.LocalDateTime;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+@Component
+public class EPOTimeRangeValidator implements ConstraintValidator<EPOTimeRange, CaseData> {
+
+    private TimeDifference maxDate;
+
+    @Override
+    public void initialize(EPOTimeRange annotation) {
+        maxDate = annotation.maxDate();
+    }
+
+    @Override
+    public boolean isValid(CaseData caseData, ConstraintValidatorContext context) {
+        LocalDateTime startDate = caseData.getDateAndTimeOfIssue();
```

Comment:
make it null safe, if entire case is validated and no dateAndTime specified it will throw NPE

Reply:
Good point, will do 👍